packages/app/src/runs/RunCard.vue

```diff
@@ -105,4 +118,21 @@ const icon = computed(() => ICON_MAP[props.gql.status!])
 
 const run = computed(() => props.gql)
 
+const relativeCreatedAt = computed(() => dayjs(new Date(run.value.createdAt!)).fromNow())
+
+const totalDuration = computed(() => dayjs.duration(run.value.totalDuration!).format('HH:mm:ss').replace(/^0+:/, ''))
+
+const tags = computed(() => {
+  const tags = props.gql.tags?.map((tag) => tag?.name).filter(Boolean) as string[]
```

Comment:
Is tags guaranteed to be defined? It doesn't seem to be nonNull in the schema, which could cause an issue for the length check below. Unless arrays are nonNull by default?

Reply:
Totally right, added fallback in [a231a47](https://github.com/cypress-io/cypress/pull/21410/commits/a231a478617321adc57a72db1e6df4e5d3b8d670)